docs/version0.33.md

```diff
@@ -35,10 +35,12 @@ The following new features and notable changes since version 0.30.0 are included
 
 ### Binaries and supported environments
 
-OpenJ9 release 0.33.0 supports OpenJDK 8, 11, 17, and 18.
+OpenJ9 release 0.33.0 and 0.33.1 supports OpenJDK 8, 11, 17, and 18.
 
 OpenJ9 Windows&reg; builds for OpenJDK 8 are now compiled with Microsoft&reg; Visual Studio 2017. The Visual Studio redistributable files included with the build are updated to match.
 
+:fontawesome-solid-pencil-alt:{: .note aria-hidden="true"} **Note:** Binaries that are labeled 0.33.1 include additional bug fixes. For more information, see the [release notes](https://github.com/eclipse-openj9/openj9/blob/master/doc/release-notes/0.33/0.33.1.md).
+
```

Comment:
Re the links to the release notes here and at the end of the topic - I see that there's a new release note for 0.33.1 but the only difference is that the new features from 0.33.0 aren't in it. So it's not clear to a reader what these 2 releases are about. I suggest keeping just the 0.33 release note, in which we use "0.33.x" (if that's what we use in the doc table of contents, for consistency) when we're talking about both releases and specific release numbers when we're talking about 1 release, eg when linking to 0.33.0 vs 0.33.1 commit history. Then the bug that was fixed can be added in the "Known issues" table so 0.33.0 users are aware of it, with a comment to say it was fixed in 0.33.1. Then it will be clear to readers what the 2 releases are about and they only have to look at 1 file to find the info.

Reply:
Updating OpenJ9 via https://github.com/eclipse-openj9/openj9/pull/15745